ports/toziuha/Toziuha Night - Dracula's Revenge.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+get_controls
+
+GAMEDIR=/$directory/ports/toziuha/
+CONFDIR="$GAMEDIR/conf/"
+
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+# Ensure the conf directory exists
+mkdir -p "$GAMEDIR/conf"
+
+# Set the XDG environment variables for config & savefiles
+export XDG_DATA_HOME="$CONFDIR"
+export SDL_GAMECONTROLLERCONFIG="$sdl_controllerconfig"
+
+cd $GAMEDIR
+
+# patch game
+if [ -f "gamedata/toziuha.pck" ]; then
+    # get toziuha.pck checksum for the Linux version from Itch.io
+    checksum=$(md5sum "gamedata/toziuha.pck" | awk '{ print $1 }')
+		# # Checksum for the Itch versio
+		if [ "$checksum" == "7a9a0252621e36cc053f1efec1a646ec" ]; then
+		$controlfolder/xdelta3 -d -s "gamedata/toziuha.pck" "patch/toziuhapatch.xdelta" "gamedata/toziuhapatched.pck" && rm "gamedata/toziuha.pck"
+		echo "Pck file from itch.io has been patched"
+    else
+	    echo "checksum does not match; wrong build/version of game"
+	fi
+	elif [ -f "gamedata/toziuha_night_dracula's_revenge.pck" ]; then
+	 # get toziuha_night_dracula's_revenge.pck checksum for Windows and Linux versions from Steam
+    checksum=$(md5sum "gamedata/toziuha_night_dracula's_revenge.pck" | awk '{ print $1 }')
+		# # Checksum for the Steam versio
+		if [ "$checksum" == "7a9a0252621e36cc053f1efec1a646ec" ]; then
```

Comment:
It's the same md5sum for itch and steam version. Is that alright ?

Reply:
It is, previously I had there different one for windows itch version.. until I realized the linux on itch and windows on steam are the same files, so the only one sticking out is the windows itch version. So I changed it to this so I dont need 2 separate patch files.